src/Compilers/CSharp/Portable/Binder/Binder_Lambda.cs

```diff
@@ -94,6 +94,14 @@ internal partial class Binder
 
                 foreach (var p in parameterSyntaxList.Value)
                 {
+                    if (p.AttributeLists.Count > 0 && syntax.Kind() == SyntaxKind.AnonymousMethodExpression)
```

Comment:
I will not sign-off on the code as is, unless attributes should be allowed on lambda parameters. 

> For all i know, attributes are allowed on lambdas.

It should take only several minutes to figure out.